### ISSUE TITLE
Create Doc For Using a Customer Airflow Image

### ISSUE THREAD

**Issue by ryw** [astronomer/docs#25]
@andscoop commented on [Tue Apr 03 2018](https://github.com/astronomerio/astronomer-ee/issues/59)

Some customers may want to lock down a version of airflow, python setuptools, etc. We need to document this process of passing in a customer docker image:tag.

**Comment by vparekh94** [comment]
Hmm - I don't think we'd need this if we had a debian based image:
https://github.com/astronomer/astronomer/issues/264

The only times customers have bought up using their own image is when it comes to using a debian based image.

I think with all the time we've spent doing alpine gymnastics, it might be worth it for myself or @andrewhharmon to take a crack at building one.

**Comment by paolaperaza** [comment]
@vparekh94 Are you ok to close this in favor of astronomer/astronomer#264? Sounds like you're increasingly moving towards building a Debian based image?

**Comment by vparekh94** [comment]
No, now that I think of it-  the option of using a debian based image is a must have going forward, but doesn't make up for good instructions on what a customer needs to do to build their own image. 

Ideally, I can take away what I learn from the debian image and write that doc.
